hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieWithTimelineServer.java

```diff
@@ -124,6 +127,22 @@ public String sendRequest(String driverHost, int port) throws RuntimeException {
       return result.toString();
     } catch (Exception ex) {
       throw new RuntimeException(ex);
+    } finally {
```

Comment:
could we use try with resources https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html 
and make it shorter?

Reply:
@vinothchandar Yes, updated the PR to address your comment.